specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/autoProvisioningAwsAccount.json

```diff
@@ -0,0 +1,216 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2019-01-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/autoProvisioningAwsAccountId": {
+      "get": {
+        "x-ms-examples": {
+          "Get auto provisioning aws accounts id for a subscription": {
+            "$ref": "./examples/AutoProvisioningAwsAccount/GetAutoProvisioningAwsAccountsIdSubscription_example.json"
+          }
+        },
+        "tags": [
+          "AutoProvisioningAwsAccounts"
+        ],
+        "description": "Exposes the auto provisioning aws accounts id of the subscription",
+        "operationId": "AutoProvisioningAwsAccountsId_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AutoProvisioningAwsAccountsIdList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      },
+      "put": {
```

Comment:
there should be a name segment after the resource type name which is autoProvisioningAwsAccountId. So it should be PUT .... /autoProvisioningAwsAccountId/{name}